spec/controllers/holds_controller_spec.rb

```diff
@@ -169,5 +170,218 @@
         expect(assigns(:bib).holdables.count).to eq 8
       end
     end
+
+    describe '#create' do
+      let(:place_hold_params) {
+        { catkey: '1',
+          pickup_library: 'UP_PAT',
+          pickup_by_date: '02/02/2020' }
+      }
+
+      let(:place_hold_response) { instance_double(HTTP::Response) }
+      let(:mock_client) do
+        instance_double(SymphonyClient, place_hold: place_hold_response)
+      end
+
+      before do
+        allow(SymphonyClient).to receive(:new).and_return(mock_client)
+      end
+
+      context 'when placing hold to a not holdable item that has no volumes' do
```

Comment:
```suggestion
      context 'when placing hold to an item that user already has a hold placed on' do
```

Also, I'm curious as to why you chose not to use the webmock approach here. Too much setup maybe?

Reply:
It might even be less setup actually but because I used stubs for the client spec and thought here controller just deals with what the client returns to it, not the request. Would like to hear your thoughts as well, I am open to ideas.

Reply:
Yup, makes sense 👍